CHANGES.md

```diff
@@ -4,6 +4,9 @@ Unreleased
 - Fixed a crash in watch mode introduced in 2.8.0 (#4118, fixes #4117,
   @rgrinberg)
 
+- Fixed wrong workspace discovery from `dune ocaml-merlin` and allow custom
+  context based on default to be used for merlin. (#4127, fixes #4125, @voodoos)
```

Comment:
Seems like the last commit should be reverted

Reply:
I forgot to drop it when rebasing, good catch.